src/settings.js

```diff
@@ -585,7 +585,7 @@ var ASYNCIFY_IMPORTS = [
   'emscripten_idb_store', 'emscripten_idb_delete', 'emscripten_idb_exists',
   'emscripten_idb_load_blob', 'emscripten_idb_store_blob', 'SDL_Delay',
   'emscripten_scan_registers', 'emscripten_lazy_load_code',
-  'wasi_unstable.fd_sync', '__wasi_fd_sync',
+  'wasi_snapshot_preview1.fd_sync', '__wasi_fd_sync',
 ];
```

Comment:
how about defining `var WASI_MODULE = "wasi_snapshot_preview1";` in this file, and then it can be added programattically here `WASI_MODULE + '.fd_sync'`.

Reply:
Hmm.. I don't really want to make it setting that can be controlled because there different ABIs are not compatible.   I'll see if I can make it global somewhere else outside of settings..

Reply:
Another option is to define it once in JS in say `modules.js`, and once in python in `shared.py`. However, I think it makes sense as an "internal setting"?